templates/civilian.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
   <meta name="viewport" content="width=device-width" />
-  <link rel="shortcut icon" href="image/favicon.ico" />
+  <link rel="shortcut icon" href="../image/favicon.ico" />
```

Comment:
Negative. `"{{ url_for('static', filename='images/favicon.ico') }}"` and `"{{ url_for('static', filename='styles/style.css') }}"`